### PR TITLE
libfixposix 0.3.0

### PR DESCRIPTION
**Pull request by ddacunha**

### CODE REVIEW

File: Library/Formula/libfixposix.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+class Libfixposix < Formula
+  env :std
```
Comment (DomT4) [inline Library/Formula/libfixposix.rb]: Why is this needed?

File: Library/Formula/libfixposix.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  desc "Thin wrapper over POSIX syscalls"
+  homepage "https://github.com/sionescu/libfixposix"
+  url "https://github.com/sionescu/libfixposix.git", :tag => "v0.3.0", :revision => "33d0a3365e9f911752a79229a2759a820892713c"
```
Comment (DomT4) [inline Library/Formula/libfixposix.rb]: Can we use an `archive` tarball or does this require submodules/etc?

File: Library/Formula/libfixposix.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  head "https://github.com/sionescu/libfixposix.git"
+  depends_on "automake" => :build
```
Comment (DomT4) [inline Library/Formula/libfixposix.rb]: Need an empty line between the `head` and first `depends_on`.

File: Library/Formula/libfixposix.rb (hunk 1 above)
Comment (ddacunha) [inline Library/Formula/libfixposix.rb]: Otherwise it won't find `autoreconf`
```
==> Checking out tag v0.3.0
==> autoreconf -i -f
Failed to execute: autoreconf
/usr/local/Library/Homebrew/debrew.rb:11:in `raise'
BuildError: Failed executing: autoreconf -i -f
```
Is there another way?

File: Library/Formula/libfixposix.rb (hunk 2 above)
Comment (ddacunha) [inline Library/Formula/libfixposix.rb]: No submodule required. I haven't found any available tarball. Could build one but don't know where to put it. Is there a way?

File: Library/Formula/libfixposix.rb (hunk 3 above)
Comment (ddacunha) [inline Library/Formula/libfixposix.rb]: Will do. Do I need to squash and re-submit a new pull request?

File: Library/Formula/libfixposix.rb (hunk 1 above)
Comment (DomT4) [inline Library/Formula/libfixposix.rb]: Add a `depends_on "autoconf" => :build` line.

File: Library/Formula/libfixposix.rb (hunk 2 above)
Comment (DomT4) [inline Library/Formula/libfixposix.rb]:
```ruby
url "https://github.com/sionescu/libfixposix/archive/v0.3.0.tar.gz"
sha256 ""
```

File: Library/Formula/libfixposix.rb (hunk 3 above)
Comment (DomT4) [inline Library/Formula/libfixposix.rb]: Squash is fine.

Comment (ddacunha) [inline Library/Formula/libfixposix.rb]: All changed  applied and squashed.

**Comment by ddacunha** [comment]
Looks like the failure on Travis happen before prior to the formula being tested:
```
The command "git reset --hard $TRAVIS_COMMIT" failed and exited with 128 during .
```

**Comment by mikemcquaid** [comment]
You can ignore that for now, it'll be picked up by our Jenkins. Sorry for the noise.

**Comment by ddacunha** [comment]
Not sure what steps I can take to address this:
```
libfixposix:
 * GitHub repository not notable enough (<10 forks, <10 watchers and <20 stars)

Error: 1 problem in 1 formula
```

**Comment by mikemcquaid** [comment]
Normally this means the repo is not notable enough to be included but given it's in Debian/Ubuntu we can make an exception for this case.

File: Library/Formula/libfixposix.rb [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  sha256 "9fda592842c28b3beb2707b908b2bdca1982c0a37572fad5cfce2ab1ba07f6d3"
+  head "https://github.com/sionescu/libfixposix.git"
+
+  depends_on "automake" => :build
+  depends_on "autoconf" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+
+  def install
+    system "autoreconf", "-i", "-f"
```
Comment (DomT4) [inline Library/Formula/libfixposix.rb]: You can happily combine `autoreconf` arguments in the latter part here, i.e. `"-fvi"`. Matches what we use elsewhere. Also has the side-effect of triggering a new bottle build since the last one didn't start for some reason :smile_cat:

Comment (ddacunha) [inline Library/Formula/libfixposix.rb]: Done :+1:

**Comment by ddacunha** [comment]
As expected Travis failed on:
```
==> audit problems

libfixposix:

 * GitHub repository not notable enough (<10 forks, <10 watchers and <20 stars)
```

**Comment by DomT4** [comment]
Once it works its way through Jenkins and the binary packages (Bottles) are produced this can be merged, per Mike's comment on the notability above.

**Comment by DomT4** [comment]
I can't pull this locally at all. Even the rebase falls apart. I'll let Mike try, maybe my `git` is having a bad evening:

```
Applying: libfixposix 0.3.0
Using index info to reconstruct a base tree...
Falling back to patching base and 3-way merge...
Auto-merging Library/Formula/libfixposix.rb
CONFLICT (add/add): Merge conflict in Library/Formula/libfixposix.rb
Failed to merge in the changes.
Patch failed at 0001 libfixposix 0.3.0
The copy of the patch that failed is found in:
   /usr/local/.git/rebase-apply/patch
```

**Comment by ddacunha** [comment]
@DomT4 What does `git status` says? Do you have a version of the file with different permissions?

**Comment by DomT4** [comment]
No unexpected changes. Can you rebase this against the master? That usually shakes off problems pulling. Mike is away at the moment so you're stuck with me trying to pull this I'm afraid :smile_cat:.

**Comment by ddacunha** [comment]
@DomT4 it's done.

**Comment by DomT4** [comment]
Thanks. Once it has passed (or rather only failed the expected element) of the Jenkins CI I'll try and pull again.

**Comment by DomT4** [comment]
Did the trick. Merged in https://github.com/Homebrew/homebrew/commit/62bd5f763ca200d709c599cdfdecd184d4d785ee. Thank you for your contribution to Homebrew @ddacunha!

**Comment by ddacunha** [comment]
Great, thanks to you guys for keeping `homebrew` going.
